Alicerce/Sources/Logging/Log+ItemStringFormatter.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  Log+ItemStringFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public struct ItemStringFormatter: LogItemFormatter {
+
+        private static let defaultFormatString = "$DHH:mm:ss.SSS$d $C$L$c $N.$F:$l - $M"
+        public let formatString: String
+        public let formatter = DateFormatter()
+        public let levelFormatter: LogItemLevelFormatter
+
+        // MARK:- Lifecycle
+
+        public init(formatString: String = ItemStringFormatter.defaultFormatString,
```

Comment:
Hum, I played around with that a little and it seems that I can only have implicit initialisers for var's, not for let's. I'd really like to maintain ```formatString``` and ```levelFormatter``` non mutable during the object's lifetime, so I think I'd prefer this initialiser having default values. Or is there something that I've not seen?